Cesium3DTiles/src/Tileset.cpp

```diff
@@ -1868,24 +1911,34 @@ static bool anyRasterOverlaysNeedLoading(const Tile& tile) {
 
 /*static*/ void Tileset::addTileToLoadQueue(
     std::vector<Tileset::LoadRecord>& loadQueue,
-    const ViewState& viewState,
+    const std::vector<ViewState>& frustums,
     Tile& tile,
-    double distance) {
+    const std::vector<double>& distances) {
   if (tile.getState() == Tile::LoadState::Unloaded ||
       anyRasterOverlaysNeedLoading(tile)) {
-    double loadPriority = 0.0;
-
-    glm::dvec3 tileDirection =
-        getBoundingVolumeCenter(tile.getBoundingVolume()) -
-        viewState.getPosition();
-    double magnitude = glm::length(tileDirection);
-    if (magnitude >= CesiumUtility::Math::EPSILON5) {
-      tileDirection /= magnitude;
-      loadPriority =
-          (1.0 - glm::dot(tileDirection, viewState.getDirection())) * distance;
+
+    glm::dvec3 boundingVolumeCenter =
+        getBoundingVolumeCenter(tile.getBoundingVolume());
+
+    double maxLoadPriority = 0.0;
+    for (size_t i = 0; i < frustums.size() && i < distances.size(); ++i) {
+      const ViewState& frustum = frustums[i];
+      const double& distance = distances[i];
+
+      glm::dvec3 tileDirection = boundingVolumeCenter - frustum.getPosition();
+      double magnitude = glm::length(tileDirection);
+
+      if (magnitude >= CesiumUtility::Math::EPSILON5) {
+        tileDirection /= magnitude;
+        double loadPriority =
+            (1.0 - glm::dot(tileDirection, frustum.getDirection())) * distance;
+        if (loadPriority > maxLoadPriority) {
+          maxLoadPriority = loadPriority;
```

Comment:
I think you want the minimum load priority. Lower priority values load sooner.

Reply:
I also didn't spot that, although, as you might remember, I once criticized that "low `priority`" means "high priority". Things like this cause bugs, period.